README.md

```diff
@@ -77,6 +77,31 @@ include a separate stylesheet in order to support Internet Explorer
 
 [service-manual-browsers]: https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices#browsers-to-test-in
 
+## Support for assistive technology
+
+GOV.UK Frontend will allow you to build services that comply with the [guidance
+in the Service Manual][service-manual-assistive-technologies].
+
+Currently, GOV.UK Frontend officially supports the following assistive technology:
+
+| Software                 | Version        | Type               | Browser                  |
+|--------------------------|----------------|--------------------|--------------------------|
+| JAWS                     | 15 or later    | screen reader      | Internet Explorer 11     |
+| ZoomText                 | 10.11 or later | screen magnifier   | Internet Explorer 11     |
+| Dragon NaturallySpeaking | 11 or later    | speech recognition | Internet Explorer 11     |
+| NVDA                     | 2016 or later  | screen reader      | Firefox (latest)         |
+| VoiceOver                | 7.0 or later   | screen reader      | Safari on iOS10 and OS X |
+
+In addition, we test that all content is accessible with keyboard only.
+
+We also aim to support high contrast mode for those users who need to adjust browser colours to increase readability. We have in the past tested this by setting custom colours in Firefox (under Preferences), and with the [High Contrast][high-contrast] plugin. Read more about [how users change colours on websites][how-users-change-colours-on-websites].
```

Comment:
Suggest linking to Firefox's docs: https://support.mozilla.org/en-US/kb/change-fonts-and-colors-websites-use

Reply:
A couple of minor things with this paragraph…

- I don't think it's just high contrast mode we support
- We mention high contrast mode but don't talk about testing with it
- I think there's reasons that users would override that don't relate to readability (e.g. high constrast gives them migraines).

How about something like…

> We aim to support [users who adjust or override the colours of websites they visit][how-users-change-colours-on-websites]. We test this by [changing colours in Firefox][firefox], by [enabling 'High Contrast' mode in Windows][windows] and by using the [High Contrast plugin for Chrome][high-contrast].

[firefox]: https://support.mozilla.org/en-US/kb/change-fonts-and-colors-websites-use
[high-contrast]: https://chrome.google.com/webstore/detail/high-contrast/djcfdncoelnlbldjfhinnjlhdjlikmph?hl=en-US
[how-users-change-colours-on-websites]: https://accessibility.blog.gov.uk/2017/03/27/how-users-change-colours-on-websites/
[windows]: https://support.microsoft.com/en-gb/help/13862/windows-use-high-contrast-mode